install-dev/upgrade/sql/1.7.7.1.sql

```diff
@@ -1 +1,3 @@
 /* PHP:ps_1771_update_customer_note(); */;
+
+UPDATE `PREFIX_currency` SET numeric_iso_code = CONCAT('0', numeric_iso_code) WHERE LENGTH(numeric_iso_code) = 2;
```

Comment:
```suggestion
UPDATE `PREFIX_currency` SET numeric_iso_code = LPAD(numeric_iso_code, 3, '0') WHERE LENGTH(numeric_iso_code) != 3;
```
What about numeric_iso_code before 10 :trollface: